### ISSUE TITLE
wrong spying behavior

### ISSUE THREAD

**Issue by kwolfy** [NagRock/ts-mockito#66]
Thank for great library!

I found unexpected behavior with properties

ts-mockito version: 2.2.5
```js
import {spy} from "ts-mockito";

class Foo {
  counter: number;
  constructor() {
    this.counter = 111;
  }
  hello(name: string) {
    return `Hello ${name} ${this.counter}!`;
  }
}

const foo: Foo = new Foo();
const mockedFoo = spy(foo);

console.log(foo.hello("world"));
/*
result: Hello world function () {
            var args = [];
            for (var _i = 0; _i < arguments.length; _i++) {
                args[_i] = arguments[_i];
            }
            var action = new MethodAction_1.MethodAction(key, args);
            _this.methodActions.push(action);
            var methodStub = _this.getMethodStub(key, args);
            methodStub.execute(args);
            return methodStub.getValue();
        }!
*/
```

**Comment by kwolfy** [comment]
I added tests that fail but shouldn't
https://github.com/kwolfy/ts-mockito/commit/c1ccbd909c3309ad1ea212ed56e4cb87b7eb2b58

**Comment by NagRock** [comment]
@kwolfy thanks for reporting and resolving this issue at once 👍 

Published as 2.2.7 version: https://github.com/NagRock/ts-mockito/releases/tag/v2.2.7
